docs/_data/nav.json5

```diff
@@ -55,6 +55,9 @@
         label: 'Custom Elements',
         path: 'elements',
         children: [
+            { label: '<hx-alert>', path: 'hx-alert' },
+            { label: '<hx-alertaction>', path: 'hx-alertaction' },
```

Comment:
we really need to document this but can we pls update `<hx-alertaction>` to <hx-alert-action> for all the instances? same for description ***note to self: need to update in documentation asap

Reply:
Will do